docs/reference/python-sdk.rst

```diff
@@ -4,49 +4,74 @@
+      waiting for it to complete, and finding the top-performing checkpoint.
+
+      The first step is to import the client module and possibly to call login():
+
+      .. code:: python
+
+         from determined.experimental import client
+
+         # We will assume that you have called `det user login`, so this is unnecessary:
+         # client.login(master=..., user=..., password=...)
+
+      The next step is to call create_experiment():
 
-As a simple example, let’s walk through the most basic workflow for creating an experiment, waiting
-for it to complete, and finding the top-performing checkpoint.
+      .. code:: python
 
-The first step is to import the client module and possibly to call login():
+         # Config can be a path to a config file or a Python dict of the config.
+         exp = client.create_experiment(config="my_config.yaml", model_dir=".")
+         print(f"started experiment {exp.id}")
 
-.. code:: python
+      The returned object is an ``Experiment`` object, which offers methods to manage the
+      experiment's lifecycle. In the following example, we simply await the experiment's completion.
 
-   from determined.experimental import client
+      .. code:: python
 
-   # We will assume that you have called `det user login`, so this is unnecessary:
-   # client.login(master=..., user=..., password=...)
+         exit_status = exp.wait()
+         print(f"experiment completed with status {exit_status}")
 
-The next step is to call create_experiment():
+      Now that the experiment has completed, you can grab the top-performing checkpoint from
+      training:
 
-.. code:: python
+      .. code:: python
 
-   # Config can be a path to a config file or a Python dict of the config.
-   exp = client.create_experiment(config="my_config.yaml", model_dir=".")
-   print(f"started experiment {exp.id}")
+         best_checkpoint = exp.list_checkpoints()[0]
+         print(f"best checkpoint was {best_checkpoint.uuid}")
 
-The returned object is an ``Experiment`` object, which offers methods to manage the experiment's
-lifecycle. In the following example, we simply await the experiment's completion.
+   .. tab::
 
-.. code:: python
+      Advanced Example
 
-   exit_status = exp.wait()
-   print(f"experiment completed with status {exit_status}")
+      **Run and Administer Experiments**
 
-Now that the experiment has completed, you can grab the top-performing checkpoint from training:
+      Visit the `det-python-sdk-demo
+      <https://github.com/determined-ai/determined-examples/tree/main/blog/python_sdk_demo>`__ to
+      learn how to run and administer experiments using the Python SDK.
 
-.. code:: python
+      .. note::
 
-   best_checkpoint = exp.list_checkpoints()[0]
-   print(f"best checkpoint was {best_checkpoint.uuid}")
+         Note: This repository contains a variety of Determined examples that are not actively
+         maintained by the Determined team. Results may vary.
```

Comment:
Idea: what if instead of stating this, we linked to a specific version of the github repo?  So that if it gets updated in the future (to work with a newer version of the python sdk) the link goes to the version we know works?